### PR TITLE
Update on the bus.json for Lagos Nigeria

### LINKED ISSUE #6243: Bus transit within Lagos
The data about bus transit in Africa's largest city needs to be added, alongside every searchable information. Bus Rapid Transit System (BRT) and LagBus which runs from Berger bustop to CMS bustop.

### PR DESCRIPTION
**Pull request by EOEboh**
It seems the id is automatically generated, for some reasons, I could not run the build. Please look into that

---
close #6243

### CODE REVIEW

File: data/transit/route/bus.json [hunk 1]
```diff
@@ -1780,6 +1780,16 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "BRT (Lagos, Nigeria)",
+      "locationSet": {"include": ["ng"]},
+      "tags": {
+        "network": "BRT",
```
Comment (LaoshuBaby) [inline data/transit/route/bus.json:1787]:
```suggestion
        "network": "Lagos Bus Rapid Transit System", 
```

File: data/transit/route/bus.json [hunk 2]
```diff
@@ -1780,6 +1780,16 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "BRT (Lagos, Nigeria)",
+      "locationSet": {"include": ["ng"]},
```
Comment (LaoshuBaby) [inline data/transit/route/bus.json:1785]:
```suggestion
      "locationSet": {"include": ["ng"]},
      "matchNames": ["Lagos BRT"],
```

**Review by LaoshuBaby — APPROVED** [review]

**Comment by LaoshuBaby** [comment]
Hey friend, congratulate for your first PR.

However I have a question that whether we use the full name as network name or not? Then add some name people will use to `matchNames`.

(I worry that too general matchName will let people live in this country wrong use this city's NSI entry, when just type a "BRT". (This also in #6257 ))

————

Also thanks to @bhousel for find the exact wikidata entry with limited hint.

**Comment by EOEboh** [comment]
Okay noted, I would effect the changes immediately. Thank you for your guidance

**Comment by EOEboh** [comment]
> Hey friend, congratulate for your first PR.
> 
> However I have a question that whether we use the full name as network name or not? Then add some name people will use to `matchNames`.
> 
> (I worry that too general matchName will let people live in this country wrong use this city's NSI entry, when just type a "BRT". (This also in #6257 ))
> 
> ————
> 
> Also thanks to @bhousel for find the exact wikidata entry with limited hinti 

> Hey friend, congratulate for your first PR.
> 
> However I have a question that whether we use the full name as network name or not? Then add some name people will use to `matchNames`.
> 
> (I worry that too general matchName will let people live in this country wrong use this city's NSI entry, when just type a "BRT". (This also in #6257 ))
> 
> ————
> 
> Also thanks to @bhousel for find the exact wikidata entry with limited hint.

I have committed the suggestions, thanks.

**Comment by arch0345** [comment]
Thanks @EOEboh and welcome!

**Pull request merged**